.github/workflows/test.yml

```diff
@@ -69,3 +69,22 @@ jobs:
       run: |
         cd prusti-contracts-test/
         cargo build
+    - name: Check Prusti binaries on Ubuntu and MacOS
+      if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+      run: |
+        # Should succeed
+        target/debug/prusti-rustc \
+          prusti-tests/tests/verify/pass/no-annotations/assert-true.rs
+        # Should fail
+        ! target/debug/prusti-rustc \
+          prusti-tests/tests/verify/fail/no-annotations/assert-false.rs
+    - name: Check Prusti binaries on Windows
+      if: matrix.os == 'windows-latest'
+      run: |
+        :: Should succeed
+        target\debug\prusti-rustc.exe \
+          prusti-tests\tests\verify\pass\no-annotations\assert-true.rs
+        :: Should fail
+        ! target\debug\prusti-rustc.exe \
+          prusti-tests\tests\verify\fail\no-annotations\assert-false.rs
+        if %errorlevel% eq 0 exit /b 1
```

Comment:
I am confused about how this works. The `!` on line 88 should negate the result of running `prusti-rustcs`. But we are in `cmd.exe` (I assume), which is why you also check if the errorlevel was 0, and fail if so? Also, `eq` should be `equ`?

Reply:
Thanks for finding this, the `!` should not be there. The `eq` worked in the past, but `==` should be a bit more readable.